anypoint-mq/2.x/modules/ROOT/pages/anypoint-mq-ack.adoc

```diff
@@ -8,8 +8,14 @@ The Anypoint MQ connector provides three acknowledgment configurations: Immediat
 == Immediate Acknowledgment
 
 Use the `IMMEDIATE` acknowledgment mode to acknowledge messages as your app consumes them, before processing them.
+When a message is acknowledged, Anypoint MQ removes it from the queue.
 
-Acknowledging the message automatically when it's received means that the message isn't redelivered if an error occurs during message processing. To handle errors without losing messages, use a dead letter queue.
+// MQ IMMEDIATE mode app restart note
+include::connectors::partial$mq-disclaimer.adoc[tag=mqImmediateAppRestart]
+
+Acknowledging the message automatically when it's received means that the message isn't redelivered if an error occurs during processing.
```

Comment:
instead of adding new line above we can edit this one - 
`Acknowledging the message automatically when it's received means it will be removed from the queue and wont get redelivered if an error occurs during processing`

Reply:
The new line = the note (in the partial) about app restart?